pywinauto/handleprops.py

```diff
@@ -145,13 +145,13 @@ def contexthelpid(handle):
 #=========================================================================
 def iswindow(handle):
     """Return True if the handle is a window"""
-    return False if handle is None else  bool(win32functions.IsWindow(handle))
+    return False if handle is None else bool(win32functions.IsWindow(handle))
 
 
 #=========================================================================
 def isvisible(handle):
     """Return True if the window is visible"""
-    return False if handle is None else  bool(win32functions.IsWindowVisible(handle))
+    return False if handle is None else bool(win32functions.IsWindowVisible(handle))

```

Comment:
Maybe `win32gui.IsWindowVisible` can be used. But I'm not sure if it can raise some exception which we don't handle here.